tests/models/mask2former/test_modeling_mask2former.py

```diff
@@ -510,3 +515,47 @@ def test_export(self):
         torch.testing.assert_close(
             eager_outputs.class_queries_logits, exported_outputs.class_queries_logits, rtol=TOLERANCE, atol=TOLERANCE
         )
+
+
+class TestMask2FormerModelInitialization(unittest.TestCase):
+    @classmethod
+    def setUpClass(cls):
+        cls.config = Mask2FormerConfig()
+        cls.model = Mask2FormerModel(cls.config)
+    
+    def test_embedding_initialization(self):
+        """Test that embeddings are initialized with std=1.0 (PyTorch default)"""
+        for name, module in self.model.named_modules():
+            if isinstance(module, torch.nn.Embedding):
+                # Skip level_embed which is intentionally initialized differently
+                if 'level_embed' in name:
+                    continue
+                std = torch.std(module.weight.data).item()
+                self.assertAlmostEqual(std, 1.0, places=1)
+    
```

Comment:
No, we should not create a new test class, just add these test methods to the `Mask2FormerModelTest`